cdc/redo/writer/writer.go

```diff
@@ -390,6 +396,50 @@ func (l *LogWriter) GetCurrentResolvedTs(ctx context.Context, tableIDs []int64)
 	return ret, nil
 }
 
+// 	DeleteAllLogs implement the DeleteAllLogs api
+func (l *LogWriter) DeleteAllLogs(ctx context.Context) error {
+	var err error
+	err = multierr.Append(err, l.rowWriter.GC(uint64(math.MaxUint32)))
+	err = multierr.Append(err, l.ddlWriter.GC(uint64(math.MaxUint32)))
```

Comment:
`math.MaxUint32` does not equal to the checked GC max ts.
Has the function `DeleteAllLogs` be tested?

Reply:
tested but with api mocked, change to a const for maxTs